README.md

```diff
@@ -22,21 +22,15 @@ npm install @getyoti/sdk-sandbox
 
 * `/path/to/your-pem-file.pem` is the path to the application pem file. It can be downloaded only once from the Keys tab in your Yoti Hub.
 
-Please do not open the pem file as this might corrupt the key and you will need to create a new application.
+* Point the Yoti client at the sandbox by setting environment variable `YOTI_CONNECT_API` to https://api.yoti.com/sandbox/v1
 
-## Profile Token Creation
+Please do not open the pem file as this might corrupt the key and you will need to create a new application.
```

Comment:
Don't need to create a new application any more:
```suggestion
Please do not open the pem file as this might corrupt the key and you will need to recreate the keys on the Yoti Hub.
```

Reply:
I'll open a separate PR for this as development is protected (this is just merging approved changes into master without release)